src/FSharp.Control.TaskSeq.Test/TestUtils.fs

```diff
@@ -95,23 +95,25 @@ type DummyTaskFactory(µsecMin: int64<µs>, µsecMax: int64<µs>) =
     /// <summary>
     /// Creates dummy tasks with a randomized delay and a mutable state,
     /// to ensure we properly test whether processing is done ordered or not.
-    /// Uses the defaults for <paramref name="µsecMin" /> and <paramref name="µsecMax" />
+    /// Uses the defaults for <paramref name="minMs" /> and <paramref name="maxMs" />
     /// with 10,000µs and 30,000µs respectively (or 10ms and 30ms).
     /// </summary>
-    new() = new DummyTaskFactory(10_000L<µs>, 30_000L<µs>)
+    new() = DummyTaskFactory(10_000L<µs>, 30_000L<µs>)
 
     /// <summary>
     /// Creates dummy tasks with a randomized delay and a mutable state,
     /// to ensure we properly test whether processing is done ordered or not.
-    /// Values <paramref name="msecMin" /> and <paramref name="msecMax" /> can be
+    /// Values <paramref name="minMs" /> and <paramref name="maxMs" /> can be
```

Comment:
Same here
```suggestion
    /// Values <paramref name="msecMin" /> and <paramref name="msecMax" /> can be
```

Reply:
was trying to put the UoM last as its arguably almost redundant in order to bring the role (min/max) of the arg to the fore.

If you're against that, fine with reverting them all as it's not like they're in anyone's face....

Reply:
It's more that `minMs` is not the same as `µsecMin`, and `minµs` looks weird. Hence the inversion.

Reply:
It's all trade-offs (and good stuff for catching me mixing up the uom...)